packages/desktop-client/src/components/settings/SimpleFINReset.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+
+import { useActions } from '../../hooks/useActions';
+import { useSimpleFinStatus } from '../../hooks/useSimpleFinStatus';
+import { ButtonWithLoading } from '../common/Button';
+import { Text } from '../common/Text';
+
+import { Setting } from './UI';
+
+export function SimpleFINResetCredential() {
+  const actions = useActions();
+  const [isSimpleFinSetupComplete, setIsSimpleFinSetupComplete] = useState<
+    boolean | null
+  >(null);
+  const [loadingSimpleFinAccounts, setLoadingSimpleFinAccounts] =
+    useState(false);
+
+  const onSimpleFinInit = () => {
+    setLoadingSimpleFinAccounts(true);
+    actions.pushModal('simplefin-init', {
+      onSuccess: () => {
+        setIsSimpleFinSetupComplete(true);
+      },
+    });
+    setLoadingSimpleFinAccounts(false);
+  };
+  const { configuredSimpleFin } = useSimpleFinStatus();
+  useEffect(() => {
+    setIsSimpleFinSetupComplete(configuredSimpleFin);
+  }, [configuredSimpleFin]);
+
+  return (
+    <Setting
+      primaryAction={
+        <ButtonWithLoading
+          loading={loadingSimpleFinAccounts}
+          disabled={!isSimpleFinSetupComplete}
+          onClick={onSimpleFinInit}
+        >
+          Reset SimpleFIN Credential
+        </ButtonWithLoading>
+      }
+    >
+      {isSimpleFinSetupComplete ? (
+        <Text>
+          <strong>Reset SimpleFIN Credential</strong> In some cases (like
+          importing from an existing database to a new instance), you may need
```

Comment:
A bit of a nit, but the other settings incorporate the bolded text into the first sentence.  Here it is acting more like a heading.  Can you reword the text?  Something like:

```
<strong>Reset SimpleFIN credential</strong> if you are having trouble accessing SimpleFIN. ...
```